website/docs/r/sddc.html.markdown

```diff
@@ -161,6 +161,8 @@ The following arguments are supported:
 
 * `num_host` - (Required) The number of hosts.
 
+* `size` - (Optional) The size of the vCenter and NSX appliances. 'large' SDDC size corresponds to a 'large' vCenter appliance and 'large' NSX appliance. 'medium' SDDC size corresponds to 'medium' vCenter appliance and 'medium' NSX appliance. Default : 'medium'.
```

Comment:
should examples be updated to include uppercase values for size?

Reply:
Done updated the size description.